packages/amplify-category-geo/src/service-stacks/deviceLocationTrackingStack.ts

```diff
@@ -96,6 +101,9 @@ export class DeviceLocationTrackingStack extends BaseStack {
 
     // set up custom params
 
+    const positionFiltering = this.parameters.get('positionFiltering')?.valueAsString;
+    const kmsKeyId = this.parameters.get('kmsKeyId')?.valueAsString;
```

Comment:
what happens in downstream flow if these are undefined ?

Reply:
When undefined, these properties are not added to the resource.